Utilities/action_web.py

```diff
@@ -106,13 +106,34 @@ def get_element(self, page, element, platform_name, dict_save_value):
         for element_yaml in arr_element:
             if element_yaml.id.__eq__(element):
                 arr_locator = element_yaml.list_locator
-                for index, loc in enumerate(arr_locator):
-                    if loc.device != platform_name:
-                        del arr_locator[index]
+                arr_locator = list(filter(
+                    lambda loc: loc.device == platform_name,
+                    arr_locator
+                ))
                 if len(arr_locator) == 1:
                     arr_locator[0].value = arr_locator[0].value.replace("{text}", text)
-                    break
-        return element_yaml
+                    return element_yaml
+    def get_element_from_data_table(self, page, table, platform_name, dict_save_value, driver, wait):
+        arr_element = page.list_element
+        if table is not None:
+            for row in table:
+                for element_yaml in arr_element:
+                    if element_yaml.id.__eq__(row["Field"]):
+                        logging.info("Verifying for %s have value %s and status %s", row["Field"], row["Value"], row["Status"])
+                        value = row["Value"].replace("\"", "")
+                        if dict_save_value is not None and value in dict_save_value.keys():
```

Comment:
we can do with 
if dict_save_value:

and same at line 147

Reply:
same above